CONTRIBUTE.md

```diff
@@ -0,0 +1,30 @@
+# Contribution guidelines for Open Zaak
+
+If you want to contribute to Open Zaak, we ask you to follow these guidelines.
+
+## Reporting bugs
+If you have encountered a bug in Open Zaak, please check if an issue already exists in the list of existing [issues](https://github.com/open-zaak/open-zaak/issues), if such an issue does not exist, you can create one [here](https://github.com/open-zaak/open-zaak/issues/new/choose). When writing the bug report, try to add a clear example that shows how to reproduce said bug.
+
+## Adding new features
+Before making making changes to the code, we advise you to first check the list of existing [issues](https://github.com/open-zaak/open-zaak/issues) for Open Zaak to see if an issue for the suggested changes already exists. If such an issue does not exist, you can create one [here](https://github.com/open-zaak/open-zaak/issues/new/choose).
```

Comment:
"Before **making making** changes to the code" 

Sorry that I haven't looked at the PR earlier